### ISSUE TITLE
mypy does not check missing required keyword-only argument

### ISSUE THREAD

**Issue by elazarg** [python/mypy#2437]
Test case:
```Python
def foo(a: int, *, b: str) -> None: pass
def bar(a: int, *x, b: str) -> None: pass

foo(1)
bar(1)
```

**Comment by gvanrossum** [comment]
I believe @sixolet just predicted this one. :-)

Looks like it's going to require an additional arg kind for x, `ARG_NAMED_REQUIRED`, to specify this.

**Comment by sixolet** [comment]
Stay tuned for a separate pull request today. Today is "break up the modo change into small ones" day.

**Comment by sixolet** [comment]
I think this is fixed on master now and we can close it.

**Comment by gvanrossum** [comment]
Fixed by #2441.
